src/components/contacts/ContactAvatar.js

```diff
@@ -75,7 +75,7 @@ const sizeConfigs = colors => ({
     textSize: 'larger',
   },
   small: {
-    dimensions: 34,
```

Comment:
and this is not used anywhere else?

Reply:
it's used in 2 other places:

- list of wallets in `BackupSection.tsx` in settings. previous sizing (34) was smaller than what we have currently in figma (36)
- `ProfileHeaderButton.js` which is the avatar in the top left corner of the wallet screen and it's getting phased out with https://github.com/rainbow-me/rainbow/pull/4106 soon

the visual difference is negligible imo but i can add a special size while we transition to `ProfileHeaderButton`'s new design if you feel like it's necessary

Reply:
Not my call, but imo we should include visual changes not related to the scope without designers approval

Reply:
right, i'll make a custom size for the previous version so it's 1:1 and leave a note about removing it when `ProfileHeaderButton` changes